crates/typst-library/src/visualize/polygon.rs

```diff
@@ -91,3 +97,55 @@ impl Layout for PolygonElem {
         Ok(Fragment::frame(frame))
     }
 }
+
+impl Reflect for PolygonElem {
+    fn describe() -> CastInfo {
+        CastInfo::Any
+    }
+
+    fn castable(_: &Value) -> bool {
+        false
+    }
+}
```

Comment:
Any particular reason for this? (Perhaps the reasoning should be written as a comment above the `impl`)

Reply:
Reflect is required for the function scope; I will add a relevant comment.

```error[E0277]: the trait bound `polygon::PolygonElem: Reflect` is not satisfied
   --> crates\typst-library\src\visualize\polygon.rs:127:1
    |
127 | #[func]
    | ^^^^^^^ the trait `Reflect` is not implemented for `polygon::PolygonElem```

Reply:
Documentation for Reflect implementation justification has been added.